### ISSUE TITLE
Bitrise installed package version is different from the package we have locally.

### ISSUE THREAD

**Issue by Josamuel** [bitrise-community/steps-cordova-archive#21]
For some reason the version of the plugin cordova-plugin-file that bitrise tries to install when we build is 6.0.1.  Our project is expecting some flavor of version 4, so it fails.  Is there a place I can edit what version of a cordova plugin bitrise is installing?  

One way around this that I attempted was to update our code's version of the plugin to 6.0.1, but for some reason I am still getting this error. 

Here is the specific error:

Failed to install 'cordova-plugin-media': CordovaError: Version of installed plugin: "cordova-plugin-file@6.0.1" does not satisfy dependency plugin requirement "cordova-plugin-file@^4.0.0". Try --force to use installed plugin as dependency.
    at /usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/plugman/install.js:557:37
    at _fulfilled (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:787:54)
    at self.promiseDispatch.done (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:816:30)
    at Promise.promise.promiseDispatch (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:749:13)
    at /usr/local/lib/node_modules/cordova/node_modules/q/q.js:509:49
    at flush (/usr/local/lib/node_modules/cordova/node_modules/q/q.js:108:17)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickCallback (internal/process/next_tick.js:180:9)
Error: Version of installed plugin: "cordova-plugin-file@6.0.1" does not satisfy dependency plugin requirement "cordova-plugin-file@^4.0.0". Try --force to use installed plugin as dependency.
cordova failed, error: exit status 1

Apologies if this is just a configuration issue I'm missing or is out of scope for this project.  Any direction would be helpful.  Thanks!

**Comment by bayssmekanique** [comment]
We ran into this problem as well.  It's because by default the step removes the platforms, and then re-adds them, which overwrites the data in the `config.xml` file.  I would recommend setting the `Should remove platforms as prepare step?` to `false`, as that will disable that portion which will likely resolve that issue.

**Comment by Josamuel** [comment]
The problem was actually our whole workflow stack.  I tried making this build configuration edit (thanks!) and then I got an error around 'ios platform already added.'  I then looked at the actual stack a bit more and read some more documentation and realized that for this particular issue, the problem was we were trying to use the latest stack (cordova 7.0.1, I believe) and our app was on cordova 6.5.0.  I chose that older stack and was able to move past the issue.

Thanks so much for your help!
